backend/qemu.pm

```diff
@@ -959,6 +959,8 @@ sub start_qemu ($self) {
         sp('device', 'usb-kbd') if $use_usb_kbd;
         sp('device', 'virtio-keyboard') if $use_virtio_kbd;
 
+        sp("device", "canokey,file=canokey") if $vars->{FIDO2};
```

Comment:
Can you add a simple unit test to confirm that the arguments are added? `t/18-backend-qemu.t` should be relatively easy to extend.

Reply:
done